### PR TITLE
adding elements in View model in JavaSwing

### PR DESCRIPTION
**Pull request by LimbertVargas**
Hi guys, I send the initial Windows elements

### CODE REVIEW

File: src/WindowCoffeMaker/WindowCoffeeMaker.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+import java.awt.event.*;
+
+public class WindowCoffeeMaker extends JFrame {
+	ImageIcon imagen = new ImageIcon("Principal.png");
+	JLabel etiqueta = new JLabel(imagen);
+	public WindowCoffeeMaker() {
+		getContentPane().add(etiqueta);
+		this.setSize(600, 450);
+	}
+	public static void main(String args[]) {
```
Comment (MadayAlcala) [inline src/WindowCoffeMaker/WindowCoffeeMaker.java:14]: missing comments on what the code does

**Comment by JosueRodriguezGarcia** [comment]
Ok Limbert already review it and I have no observations.

**Comment by MadayAlcala** [comment]
I think the code is ready to upload.

**Comment by alszla** [comment]
Hmmm

**Comment by alszla** [comment]
XD. Everything is OK

File: src/WindowCoffeMaker/WindowCoffeeMaker.java (hunk 1 above)
Comment (LimbertVargas) [reply src/WindowCoffeMaker/WindowCoffeeMaker.java:14]: ok, dude, I solve it

**Pull request merged**